### PR TITLE
Fix doc references

### PR DESCRIPTION
**Pull request by NeQuissimus**
Some of the references/links were dead. Fixed those.
Also I played with a colour scheme a little bit. (using https://coolors.co/ed2124-a62639-a29c9b-251605-c57b57)

### CODE REVIEW

**Comment by NeQuissimus** [comment]
Fixed formatting and deployed to http://nequissimus.com/scalaz/

File: build.sbt [hunk 1]
```diff
@@ -99,5 +99,15 @@ lazy val example = project.module
     micrositeGitHostingUrl := "https://github.com/scalaz/scalaz",
     micrositeGithubOwner := "scalaz",
     micrositeGithubRepo := "scalaz",
-    micrositeFavicons := Seq(microsites.MicrositeFavicon("favicon.png", "512x512"))
+    micrositeFavicons := Seq(microsites.MicrositeFavicon("favicon.png", "512x512")),
+    micrositePalette := Map(
+      "brand-primary"   -> "#ED2124",
+      "brand-secondary" -> "#251605",
+      "brand-tertiary"  -> "#A62639",
```
Comment (hrhino) [inline build.sbt]: I think this doesn't contrast very well with the scalaz logo in the upper right.

**Comment by NeQuissimus** [comment]
How about this?

**Comment by NeQuissimus** [comment]
I am not much of a graphics guy either, I just wanted to change away from the default colours :)

**Pull request merged**